src/main/routes/config.ts

```diff
@@ -44,6 +30,26 @@ async function getAllowedNamespaces(cluster: Cluster) {
   }
 }
 
+async function getAllowedResources(cluster: Cluster) {
+  // TODO: auto-populate all resources dynamically
+  const resources = [
```

Comment:
For now we probably could just list all resources here which are mapped to the sidebar.. It would allow truly dynamic sidebar menu that follows RBAC rules.